tests/console/commands/test_init.py

```diff
@@ -993,6 +993,24 @@ def test_validate_package_invalid(name: str) -> None:
         assert InitCommand._validate_package(name)
 
 
+@pytest.mark.parametrize(
+    "author",
+    [
+        # As MacOS saves "é" — 65 CC 81
+        "José Duarte",
+        # As written by a MacOS keyboard — C3 A9
+        "José Duarte",
```

Comment:
It seems like both strings are the same (C3 A9). The test succeeds even after reverting your change.